src/OrchardCore.Cms.Web/Program.cs

```diff
@@ -1,24 +1,20 @@
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
 using OrchardCore.Logging;
 
-namespace OrchardCore.Cms.Web
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Host.UseNLogHost();
+
+builder.Services.AddOrchardCms().AddSetupFeatures("OrchardCore.AutoSetup");
+
+var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
```

Comment:
Starting with .NET 6 the developer exception page is configured by default for development environments

[https://docs.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?view=aspnetcore-6.0](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?view=aspnetcore-6.0)

![image](https://user-images.githubusercontent.com/68876423/176704916-4413cdf0-5e79-4011-a198-6a3e3d739d3f.png)